src/core.lisp

```diff
@@ -1,3 +1,46 @@
 ;;;; Rasterization renderer.
 
 (in-package #:cl-raster/core)
+
+(ql:quickload :3d-vectors)
+(use-package :3d-vectors)
+
+(defun project-point-to-camera (camera point)
+  (let* ((to-point-vector (v- (camera-get-center camera) point))
+         (center-to-canvas-vector (v/ to-point-vector (v. to-point-vector (camera-get-direction camera))))
+         (in-canvas-vector (v- center-to-canvas-vector (camera-get-direction camera)))
+         (x-canvas-coordinate (v. (camera-get-x-vector camera) in-canvas-vector))
+         (y-canvas-coordinate (v. (camera-get-y-vector camera) in-canvas-vector)))
+    (list (round (* (/ (1+ x-canvas-coordinate) 2) (camera-get-width camera)))
+          (round (* (/ (1+ y-canvas-coordinate) 2) (camera-get-height camera))))))
+
+(defun triangle-contains (triangle point)
+  (let ((a-to-point (v- point (first triangle)))
+        (b-to-point (v- point (second triangle)))
+        (c-to-point (v- point (third triangle)))
+        (bc-perpendicular (vrot2 (v- (second triangle) (third triangle)) pi/2))
```

Comment:
There is no variable with name `pi/2`. There is, however, a constant named `pi`. You should use `(/ pi 2)`